examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -527,6 +541,18 @@ static void InitServer(intptr_t context)
     SetupInitialLevelControlValues(/* endpointId = */ 2);
 }
 
+static void InitOTARequestor(void)
+{
+#if CONFIG_ENABLE_OTA_REQUESTOR
+    SetRequestorInstance(&gRequestorCore);
+    gRequestorCore.SetServerInstance(&Server::GetInstance());
+    gRequestorCore.SetOtaRequestorDriver(&gRequestorUser);
+    gImageProcessor.SetOTADownloader(&gDownloader);
+    gDownloader.SetImageProcessorDelegate(&gImageProcessor);
+    gRequestorCore.SetBDXDownloader(&gDownloader);
+#endif
```

Comment:
Too many ifdefs in the application? :-|

Reply:
I removed the one which was guarding the header files.